src/main/resources/lib/hudson/test/non-passed-test.jelly

```diff
@@ -25,19 +25,19 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <st:documentation>
-    Display link to the failed test.
+    Display link to the test.
```

Comment:
Ditto, for renaming a reusable tag. There appears to be at least one[1] external use of this.

[1] https://github.com/jenkinsci/mentor-questa-vrm-plugin/blob/04079b04117d351878e1be16d19c79fa38ed2568/src/main/resources/com/mentor/questa/vrm/jenkins/QuestaVrmRegressionResult/body.jelly#L45